src/lib/constants.js

```diff
@@ -33,6 +33,38 @@ export const colorLabels = {
   popup_text: "Popup Text"
 };
 
+export const advancedColorLabels = {
```

Comment:
Just for the record: I verified that this list is complete. Every color listed here exists, and together with the existing `colorLabels` definition, every non-deprecated color property that Firefox [supports](https://searchfox.org/mozilla-central/rev/8ccea36c4fb09412609fb738c722830d7098602b/toolkit/components/extensions/schemas/theme.json#101-271) is covered.